LPS-74764 remove trailing whitespace

gamerson opened the pull request.

liferay-continuous-integration commented:
Pull request test invoked at http://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest(master).

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest(master)/2474/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-18.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/2474/jenkins-report.html">here</a>.</b></p>

gamerson commented:
hey @brunobasto  I made a few changes to fix some of the failling tests.  however, there is a new failing test that I wanted to see if you could take a look at what we are seeing a different.

Here is the failure:

```
com.liferay.project.templates.ProjectTemplatesTest > testBuildTemplateSoyPortlet FAILED
    org.junit.ComparisonFailure: Bundle /var/folders/zl/wtp3csp12zz06vwxlqhm_0z00000gn/T/junit5011847632853450641/gradle/foo/build/libs/com.liferay.test-1.0.0.jar and /var/folders/zl/wtp3csp12zz06vwxlqhm_0z00000gn/T/junit5011847632853450641/maven/foo/target/foo-1.0.0.jar do not match expected:<[]> but was:<[CHANGED    MANIFEST   <manifest>
     REMOVED    HEADER     Provide-Capability:osgi.service;objectClass:List<String>="com.liferay.portal.kernel.portlet.bridges.mvc.MVCRenderCommand",osgi.service;objectClass:List<String>="javax.portlet.Portlet",liferay.resource.bundle;bundle.symbolic.name="com.liferay.test";resource.bundle.base.name="content.Language"
     ADDED      HEADER     Provide-Capability:osgi.webresource;osgi.webresource=foo;version:Version="1.0.0",osgi.service;objectClass:List<String>="com.liferay.portal.kernel.portlet.bridges.mvc.MVCRenderCommand",osgi.service;objectClass:List<String>="javax.portlet.Portlet",liferay.resource.bundle;bundle.symbolic.name="com.liferay.test";resource.bundle.base.name="content.Language"
     ADDED      HEADER     Web-ContextPath:/foo-1.0.0
    ]>
        at org.junit.Assert.assertEquals(Assert.java:115)
        at com.liferay.project.templates.ProjectTemplatesTest._testBundlesDiff(ProjectTemplatesTest.java:1897)
        at com.liferay.project.templates.ProjectTemplatesTest._buildProjects(ProjectTemplatesTest.java:1597)
        at com.liferay.project.templates.ProjectTemplatesTest._buildProjects(ProjectTemplatesTest.java:1574)
        at com.liferay.project.templates.ProjectTemplatesTest.testBuildTemplateSoyPortlet(ProjectTemplatesTest.java:1081)
```

You can see this by just running: 
`gw test --tests "*testBuildTemplateSoyPortlet"`

gamerson commented:
ci:stop

liferay-continuous-integration commented:
<html><h1>Some tests FAILED.</h1><p>Build Time: 2 minutes 8 seconds 341 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/563c9ada51c787403164cc8e907ed5c5431ac83a">563c9ada51c787403164cc8e907ed5c5431ac83a</a></p>0 out of 1 jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest(master)/2474/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li></ul></div><h5>For more details click <a href="https://test-1-18.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/2474/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest(master)/2474/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><pre><code>Build was aborted</code></pre></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">here</a>.</h4></html>

gamerson commented:
actually, nevermind! it passed the 2nd time I ran it.  Sending to brian
